vmm/src/lib.rs

```diff
@@ -2064,8 +2068,7 @@ impl Vmm {
         &mut self,
         config_json: String,
     ) -> std::result::Result<(), VmmActionError> {
-        let body: Chunk = Chunk::from(config_json);
-        let vmm_config = match serde_json::from_slice::<VmmConfig>(&body) {
+        let vmm_config = match serde_json::from_slice::<VmmConfig>(config_json.as_bytes()) {
```

Comment:
Could you do something like:
```
serde_json::from_slice::<VmmConfig>(config_json.as_bytes()).unwrap_or_else(|e| {
    error!("Invalid json: {}", e);
    process::exit(i32::from(FC_EXIT_CODE_INVALID_JSON));
}
```
? It feels slightly more expressive (i.e. rather than having to parse the match, you immediately see that we take the value of the config_json, propagating any errors generated by from_slice in some way.

Reply:
Done